### ISSUE TITLE
[FEATURE] Mood tracker para el club

### ISSUE THREAD

**Issue by RafaelMoralesV** [exdevutem/discord_bot#5]
**Describa la solución que le gustaría**
El club no tiene ninguna forma de trackear el mood de sus integrantes!
Tengo un par de ideas, pero necesito refinarlas
Idealmente tiene que ser simple en términos de preguntas, relativamente superficiales para no incomodar a nadie, y guardar esta información.
Debe poder solicitar esta info a **los miembros del club solamente**!

**Describe alternativas que hayas considerado**
Tradicionalmente cuando preguntabamos esta info, solíamos vincular las respuestas de las personas con su identidad, además de visibilizar todas las respuestas en un canal particular de Slack.
No estoy seguro si esa sería la forma correcta de continuar. Quizás sería mejor un formato anónimo y ver patrones generales del club, más allá de preguntas en específicas.

Necesito opiniones del club!

**Comment by XhrdTLS** [comment]
Hacer una especie de escala 1 - 5 y a una hora específica mostrar como el puntaje del mood del club esa semana, onda "Está semana el club tiene un mood de 3.8 (también podría haber un comentario aquí de típica frase de página cringe xD)

La escala puede ser 1 mal y 5 muy bien, igual considerando solo una pregunta, también considerar el escribir el porqué, y el de los proyectos con preguntas o también alternativas idk ahí

**Comment by RafaelMoralesV** [comment]
> Hacer una especie de escala 1 - 5 y a una hora específica mostrar como el puntaje del mood del club esa semana, onda "Está semana el club tiene un mood de 3.8 (también podría haber un comentario aquí de típica frase de página cringe xD)
> 
> La escala puede ser 1 mal y 5 muy bien, igual considerando solo una pregunta, también considerar el escribir el porqué, y el de los proyectos con preguntas o también alternativas idk ahí

similar a lo que teniamos antes en slack? onda 3 preguntas:
- como te sientes hoy
- que has hecho desde la reunion anterior
- que es lo que te gustaría tener para la siguiente reunion

algo así?

Igual, me gustaría que la primera pregunta pudiera ser anónima, pero no sé si sería útil para los encargados de esa manera.

**Comment by ChinoCayul** [comment]
Puedes dejarlo a preferencia en si es anónima, - [ ] así como un tipo de check " □ anónimo", hay ocasiones igual en las que alguno prefiere contar las cosas sin decir quien es necesariamente

**Comment by XhrdTLS** [comment]
> > Hacer una especie de escala 1 - 5 y a una hora específica mostrar como el puntaje del mood del club esa semana, onda "Está semana el club tiene un mood de 3.8 (también podría haber un comentario aquí de típica frase de página cringe xD)
> > 
> > La escala puede ser 1 mal y 5 muy bien, igual considerando solo una pregunta, también considerar el escribir el porqué, y el de los proyectos con preguntas o también alternativas idk ahí
> 
> similar a lo que teniamos antes en slack? onda 3 preguntas:
> - como te sientes hoy
> - que has hecho desde la reunion anterior
> - que es lo que te gustaría tener para la siguiente reunion
> 
> algo así?
> 
> Igual, me gustaría que la primera pregunta pudiera ser anónima, pero no sé si sería útil para los encargados de esa manera.

Sí, osea como decían colocar *!a "respuesta"* por ejemplo para que sea anónima, ayudaría a que los que quieran sea anónima, ahora habría que ver por la implementación, porque si solo uno/a marca anónima, se pierde el anonimato XD, sino hacer una poll al respecto sería lo más 'optimo', porque también hay que recordar que las weeklys las íbamos a dejar presenciales 👀, ¿o eso también sería algo que hay que ver?

**Comment by RafaelMoralesV** [comment]
> Puedes dejarlo a preferencia en si es anónima, - [ ] así como un tipo de check " □ anónimo", hay ocasiones igual en las que alguno prefiere contar las cosas sin decir quien es necesariamente

si, es algo que consideré. Quizás intente prototiparlo y vemos que tal resulta.

> Sí, osea como decían colocar _!a "respuesta"_ por ejemplo para que sea anónima, ayudaría a que los que quieran sea anónima, ahora habría que ver por la implementación, porque si solo uno/a marca anónima, se pierde el anonimato XD, 

La idea es que el bot te hable al dm ps, sería super tonto de otra forma kajsndaksjd

>sino hacer una poll al respecto sería lo más 'optimo', porque también hay que recordar que las weeklys las íbamos a dejar presenciales eyes, ¿o eso también sería algo que hay que ver?

Lo de las weeklies presenciales no lo recordaba, pensé que el plan era que fueran en discord; las estuvimos haciendo presenciales de forma extraordinaria este ultimo tiempo
